tests/backend/topology/test_ring.py

```diff
@@ -11,9 +11,9 @@
 
```

Comment:
It'd be better to just parameterize the `static` variable than making it a fixture :+1: